tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_raw.cpp

```diff
@@ -41,35 +41,45 @@ TEST (nnstreamer_nnfw_runtime_raw_functions, open_close_00_n)
   EXPECT_NE (ret, 0);
 }
 
-/**
- * @brief Test nnfw subplugin with successful open/close
- */
-TEST (nnstreamer_nnfw_runtime_raw_functions, open_close_01_n)
+static
+gboolean gen_tf_prop (GstTensorFilterProperties *prop)
 {
-  int ret;
-  void *data = NULL;
-  gchar *test_model;
+  gchar *model_file;
+  gchar *meta_file;
   gchar *model_path;
   const gchar *root_path = g_getenv ("NNSTREAMER_BUILD_ROOT_PATH");
 
-  ASSERT_NE (root_path, nullptr);
+  g_return_val_if_fail (root_path != nullptr, FALSE);
 
   /** nnfw needs a directory with model file and metadata in that directory */
   model_path = g_build_filename (root_path, "tests", "test_models", "models",
       NULL);
-  GstTensorFilterProperties prop = {
-    .fwname = "nnfw",
-    .fw_opened = 0,
-    .model_file = model_path,
-  };
 
-  test_model = g_build_filename (model_path, "add.tflite", NULL);
-  ASSERT_TRUE (g_file_test (test_model, G_FILE_TEST_EXISTS));
-  g_free (test_model);
+  meta_file = g_build_filename (model_path, "metadata", "MANIFEST", NULL);
+  g_return_val_if_fail (g_file_test (meta_file, G_FILE_TEST_EXISTS), FALSE);
+  g_free (meta_file);
+
+  model_file = g_build_filename (model_path, "add.tflite", NULL);
+  g_return_val_if_fail (g_file_test (model_file, G_FILE_TEST_EXISTS), FALSE);
```

Comment:
Missing `g_free (meta_file);` ?

Reply:
line 59 and 63 will report svace issue (no-free function: meta_file and model_file)

Reply:
Ah, What I meant is line 64: `g_free (model_path)` --> `g_free (model_file)`

Reply:
updated.